src/ui-lib/sass/05-components/03-organisms/_list-2cols.scss

```diff
@@ -0,0 +1,23 @@
+.grav-c-list-2cols {
```

Comment:
AHA! good effin point. so the `list-two-columns` is for text, while this is for blocks... suggestions @franz899 ?

Reply:
They solve a similar problem, but they can't be merged into one class. The main difference is that one presents the element in a vertical fashion, without caring about alignment of the elements, where the other does it in an ordered horizontal one.

We could rename them according to how they display content on the screen, but we still have to create a modifier for: `max-width: gravy-breakpoint(medium);` used in `grav-c-list-two-columns`.

Something like `grav-c-list-aligned` and `grav-c-list-unaligned`.

Reply:
Hmm... so, `grav-c-list-two-columns` is _only_ being used on the homepage to lay out the "areas we are passionate about".

Those are just `<li>`s behind the scenes, so we could use flexbox to achieve the same thing (I know the items would appear in a different order, but for that particular content I don't think it matters). That would make `grav-c-list-two-columns` redundant and we could just have `grav-c-list-2cols`.

Reply:
Ok, after chatting to @franz899 I realise there are more difference between the two. I therefore second his suggestion of finding class names that represent what the component _represents_ rather than how it works.

I'm not convinced by `aligned` and `unaligned` though - there are many kinds of alignment, so I don't think those names convey what they do.

How about something like `grav-c-location-list` and `grav-c-indented-text-columns`?

Reply:
after chatting again with @franz899 and @c1rrus we have settled for `grav-c-two-columns-text` and `grav-c-two-columns-block`